tests/contrib/aws_lambda/test_aws_lambda.py

```diff
@@ -45,7 +47,7 @@ def setup():
 
 
 @pytest.mark.parametrize("customApmFlushDeadline", [("-100"), ("10"), ("100"), ("200")])
-@pytest.mark.snapshot()
+@pytest.mark.snapshot
```

Comment:
@pytest.mark.snapshot and @pytest.mark.snapshot() are interchangeable

Reply:
Thanks, I removed it since I realized some of them had it, and some didn't, so I just wanted to keep consistency.